src/components/ContentItem/ContentItem.stories.js

```diff
@@ -175,11 +175,118 @@ const successState = () => ({
   `,
 });
 
+const withCustomForwardIcon = () => ({
+  components: {
+    ContentItem,
+    DocumentFilledIcon,
+    StorybookMobileDeviceSimulator,
+    ClockIcon,
+  },
+  props: {
+    device: {
+      default: select('Simulated Mobile Device', [...availableDevices], availableDevices[0]),
+    },
+    disabled: {
+      default: boolean('Is Disabled?', false),
+    },
+    hasForwardButton: {
+      default: boolean('Has Forward Button?', true),
+    },
+    success: {
+      default: boolean('Has Success forward icon?', true),
+    },
+    title: {
+      default: text('Title', 'Example Title'),
+    },
+    metaText: {
+      default: text('Meta text', 'Example Metatext'),
+    },
+  },
+  methods: {
+    onClick: action('Click!'),
+  },
+  template: `
+    <div style="margin: 10px 50px 10px 50px;">
+      <h2><strong>ContentItem:</strong>&nbsp;A list item with success state, it does not redirect anywhere, it just provides success/completed information.</h2>
+      <hr>
+      <StorybookMobileDeviceSimulator :device="device">
+        <ContentItem :title="title"
+                     :meta-text="metaText"
+                     :has-forward-button="hasForwardButton"
+                     no-border
+                     :disabled="disabled"
+                     :success="success"
+                     @click="onClick"
+        >
+          <DocumentFilledIcon/>
+          <template v-slot:forward-icon>
```

Comment:
(nit) there's no need for this `<template>` wrapper here, instead: please just add the `v-slot` attribute to the `<ClockIcon/>` element: 

```js
<DocumentFilledIcon/>
<ClockIcon v-slot:forward-icon/>
```